packages/ui/src/mixins.js

```diff
@@ -46,6 +46,13 @@ export const typography = (level) => {
       color: ${colors.F2};
       font-weight: 400;
     `,
+
+    p3: css`
+      font-size: 12px;
+      line-height: 22px;
+      color: rgba(0, 0, 0, .65);
+      font-weight: normal;
+    `
```

Comment:
Is this something in the design system?

If it's a repeated styling, I'm ok to add it to this typography mixin, but just let you know all the definitions here were aligned with our UI designers. You may have a look of what we have already defined in the UI styles guide: https://lanhuapp.com/url/Q8S5u